kolibri/auth/api_urls.py

```diff
@@ -1,9 +1,6 @@
 from rest_framework import routers
 
-from .api import (
-    ClassroomViewSet, DeviceOwnerViewSet, FacilityUserViewSet, FacilityViewSet,
-    LearnerGroupViewSet, MembershipViewSet, RoleViewSet
-)
+from .api import ClassroomViewSet, DeviceOwnerViewSet, FacilityUserViewSet, FacilityViewSet, LearnerGroupViewSet, MembershipViewSet, RoleViewSet
```

Comment:
This was auto-updated based on the updated linting rules. Change here was to rename file to api_urls.py for consistency.